diff -Nurd etr-0.8.1/configure.ac etr-0.8.1/configure.ac
--- etr-0.8.1/configure.ac	2021-08-29 05:50:51.338244147 +0300
+++ etr-0.8.1/configure.ac	2021-08-29 05:51:03.434363966 +0300
@@ -42,33 +42,7 @@
 AC_FUNC_MALLOC
 AC_CHECK_FUNCS([floor getcwd memmove memset mkdir pow sqrt strchr strdup])
 
-PKG_CHECK_MODULES([SFML_SYSTEM], [sfml-system >= 2.4],
-[
-  CPPFLAGS="${CPPFLAGS} $SFML_SYSTEM_CFLAGS"
-  LIBS="${LIBS} $SFML_SYSTEM_LIBS"
-],
-[AC_MSG_ERROR([sfml-system not found])])
-
-PKG_CHECK_MODULES([SFML_AUDIO], [sfml-audio >= 2.4],
-[
-  CPPFLAGS="${CPPFLAGS} $SFML_AUDIO_CFLAGS"
-  LIBS="${LIBS} $SFML_AUDIO_LIBS"
-],
-[AC_MSG_ERROR([sfml-audio not found])])
-
-PKG_CHECK_MODULES([SFML_WINDOW], [sfml-window >= 2.4],
-[
-  CPPFLAGS="${CPPFLAGS} $SFML_WINDOW_CFLAGS"
-  LIBS="${LIBS} $SFML_WINDOW_LIBS"
-],
-[AC_MSG_ERROR([sfml-window not found])])
-
-PKG_CHECK_MODULES([SFML_GRAPHICS], [sfml-graphics >= 2.4],
-[
-  CPPFLAGS="${CPPFLAGS} $SFML_GRAPHICS_CFLAGS"
-  LIBS="${LIBS} $SFML_GRAPHICS_LIBS"
-],
-[AC_MSG_ERROR([sfml-graphics not found])])
+LIBS="${LIBS} -lsfml-system -lsfml-audio -lsfml-window -lsfml-graphics"
 
 AC_CHECK_LIB([GLU], [gluBeginCurve],
 [
