diff -Nurd physfs-2.0.2/archivers/wad.c physfs-2.0.2-/archivers/wad.c
--- physfs-2.0.2/archivers/wad.c	2011-02-18 23:17:31.000000000 +0200
+++ physfs-2.0.2-/archivers/wad.c	2012-10-05 16:47:03.105547992 +0300
@@ -245,9 +245,6 @@
     PHYSFS_uint32 fileCount;
     PHYSFS_uint32 directoryOffset;
     WADentry *entry;
-    char lastDirectory[9];
-
-    lastDirectory[8] = 0; /* Make sure lastDirectory stays null-terminated. */
 
     BAIL_IF_MACRO(!wad_open(name, forWriting, &fh, &fileCount,&directoryOffset), NULL, 0);
     info->entryCount = fileCount;
