Upstream-Status: Inappropriate

diff --git a/configure.ac b/configure.ac
index e7289e51e5..673e254ed7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -721,6 +721,8 @@ dnl Defaults to override with host specific values
 HOST_PATH_SEPARATOR=":"
 HOST_DIR_SEPARATOR="/"
 
+AC_DEFINE([ALWAYS_ROOT], [1], [Can execute program as root - OpenEmbedded])
+
 dnl Settings specific to host OS
 case "$host_os" in
 
diff --git a/meson.build b/meson.build
index dab9d6d93c..3111fcec4f 100644
--- a/meson.build
+++ b/meson.build
@@ -328,9 +328,10 @@ else
     threads='pthread'
   endif
 
+  priv_conf_data.set('ALWAYS_ROOT', 1)
+
   if host_system == 'haiku'
     net_dep = c_compiler.find_library('network')
-    priv_conf_data.set('ALWAYS_ROOT', 1)
   else
     net_dep = []
   endif
