Allow root to run freeciv

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>

diff --git a/configure.ac b/configure.ac
index b65af702c1..e1a47692d1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -663,6 +663,8 @@ fi
 dnl Defaults to override with host specific values
 HOST_PATH_SEPARATOR=":"
 
+AC_DEFINE([ALWAYS_ROOT], [1], [Can execute program as root - OpenEmbedded])
+
 dnl Settings specific to host OS
 case "$host_os" in
 
