diff --git a/configure.ac b/configure.ac
index e7651bea3e..a36dad82b1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -658,6 +658,15 @@ if test "x$XMLREGISTRY" = "xyes" ; then
   AC_DEFINE([FREECIV_HAVE_XML_REGISTRY], [1], [Build xml-backend for registry])
 fi
 
+dnl Dummy configure option for OpenEmbedded build of older branches
+dnl to avoid unrecognised option error
+AC_ARG_WITH([qtver],
+  AS_HELP_STRING([--with-qtver], [which Qt version to build against [qt5]]),
+[case "${withval}" in
+ qt5|Qt5) ;;
+ *) AC_MSG_ERROR([bad value ${enableval} for --with-qtver]) ;;
+esac])
+
 dnl Set debug flags supported by compiler
 EXTRA_DEBUG_CFLAGS=""
 EXTRA_DEBUG_CXXFLAGS=""
