Fix build with latest Qt

Upstream-Status: Submitted

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>

diff --git a/client/gui-qt/canvas.cpp b/client/gui-qt/canvas.cpp
index 4c3ac0c03f..9e230f4c16 100644
--- a/client/gui-qt/canvas.cpp
+++ b/client/gui-qt/canvas.cpp
@@ -1,4 +1,4 @@
-/**********************************************************************
+/***********************************************************************
  Freeciv - Copyright (C) 1996-2005 - Freeciv Development Team
    This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
@@ -18,6 +18,7 @@
 // Qt
 #include <QFontMetrics>
 #include <QPainter>
+#include <QPainterPath>
 
 // qt-client
 #include "canvas.h"
diff --git a/client/gui-qt/dialogs.cpp b/client/gui-qt/dialogs.cpp
index b736908bb0..32501f2bbb 100644
--- a/client/gui-qt/dialogs.cpp
+++ b/client/gui-qt/dialogs.cpp
@@ -24,6 +24,7 @@
 #include <QMessageBox>
 #include <QMouseEvent>
 #include <QPainter>
+#include <QPainterPath>
 #include <QRadioButton>
 #include <QRect>
 #include <QSignalMapper>
