From 84ab712dfa56afbec12d0d96332d1aa1cc7ea399 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 26 Aug 2018 21:39:47 +0200
Subject: [PATCH] Find sdl CFLAGS with pkg-config - sdl-config is not usable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>

diff -Nurd wesnoth-1.16.2/CMakeLists.txt wesnoth-1.16.2/CMakeLists.txt
--- wesnoth-1.16.2/CMakeLists.txt	2022-01-16 20:10:36.000000000 +0200
+++ wesnoth-1.16.2/CMakeLists.txt	2022-02-15 20:53:25.921752194 +0200
@@ -81,8 +81,8 @@
 if(NOT WIN32)
 	# needed to get some SDL2 defines in... (as of rev31694 -D_GNU_SOURCE=1 is required!)
 	if(NOT MINGW)
-		set(SDL2_CONFIG "sdl2-config" CACHE STRING "Path to sdl2-config script")
-		exec_program(${SDL2_CONFIG} ARGS "--cflags" OUTPUT_VARIABLE SDL2_CFLAGS)
+                set(SDL2_CONFIG "pkg-config" CACHE STRING "Path to sdl2-config script")
+                exec_program(${SDL2_CONFIG} ARGS "sdl2 --cflags" OUTPUT_VARIABLE SDL2_CFLAGS)
 		add_definitions(${SDL2_CFLAGS})
 	else()
 		# equivalent to sdl2-config --cflags --libs
