From 84ab712dfa56afbec12d0d96332d1aa1cc7ea399 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 26 Aug 2018 21:39:47 +0200
Subject: [PATCH] Find sdl CFLAGS with pkg-config - sdl-config is not usable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index becfff5..c1f477e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,8 +88,8 @@ find_package(X11)
 
 # needed to get some SDL2 defines in... (as of rev31694 -D_GNU_SOURCE=1 is required!)
 if(NOT MINGW)
-	set(SDL2_CONFIG "sdl2-config" CACHE STRING "Path to sdl2-config script")
-	exec_program(${SDL2_CONFIG} ARGS "--cflags" OUTPUT_VARIABLE SDL2_CFLAGS)
+	set(SDL2_CONFIG "pkg-config" CACHE STRING "Path to sdl-config script")
+	exec_program(${SDL2_CONFIG} ARGS "sdl2 --cflags" OUTPUT_VARIABLE SDL2_CFLAGS)
 	add_definitions(${SDL2_CFLAGS})
 else()
 	# equivalent to sdl2-config --cflags --libs
-- 
2.30.2

