From 5440f7ed37c8f3b170fda2a3b9e144cbb1291383 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 26 Aug 2018 21:59:24 +0200
Subject: [PATCH] Do not do the ar/ranlib configure dance - it won't work in
 oe-cross environment
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 CMakeLists.txt | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1026001..15ad0bb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -372,25 +372,6 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 	endif(ENABLE_LTO)
 endif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 
-# set CMAKE_AR and CMAKE_RANLIB to use LTO-enabled variants if LTO is enabled
-if(ENABLE_LTO)
-	MESSAGE("Using gcc-ar and gcc-ranlib")
-	find_program(LTO_AR NAMES gcc-ar)
-	find_program(LTO_RANLIB NAMES gcc-ranlib)
-	set(CMAKE_AR "${LTO_AR}" CACHE STRING "Supports LTO" FORCE)
-	set(CMAKE_RANLIB "${LTO_RANLIB}" CACHE STRING "Supports LTO" FORCE)
-else()
-	MESSAGE("Using ar, ranlib, and default linker")
-	find_program(NON_LTO_AR NAMES ar)
-	find_program(NON_LTO_RANLIB NAMES ranlib)
-	set(CMAKE_AR "${NON_LTO_AR}" CACHE STRING "Does not support LTO" FORCE)
-	set(CMAKE_RANLIB "${NON_LTO_RANLIB}" CACHE STRING "Does not support LTO" FORCE)
-endif()
-MARK_AS_ADVANCED(LTO_AR LTO_RANLIB NON_LTO_AR NON_LTO_RANLIB)
-
-# add in extra flags
-set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE} ${LINK_EXTRA_FLAGS_CONFIG} ${LINK_EXTRA_FLAGS_RELEASE}")
-
 # clean the pgo data
 set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES "${CMAKE_SOURCE_DIR}/pgo_data/")
 
-- 
2.14.4

