From 27c44b052ca69873046f32631b3ccbe1d6b11dd3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 25 Oct 2020 17:47:51 +0100
Subject: [PATCH] use pkg-config to gather sdl information
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 configure | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/configure b/configure
index 36a857e..e69e39c 100755
--- a/configure
+++ b/configure
@@ -3735,14 +3735,13 @@ fi
 # Setup SDL specifics for SDL based backends
 #
 if test "$_sdl" = auto ; then
-	find_sdlconfig
-	append_var SDL_CFLAGS "`$_sdlconfig --prefix="$_sdlpath" --cflags`"
+	append_var SDL_CFLAGS "`pkg-config --cflags sdl2`"
 	if test "$_static_build" = yes ; then
 		append_var SDL_LIBS "`$_sdlconfig --prefix="$_sdlpath" --static-libs`"
 	else
-		append_var SDL_LIBS "`$_sdlconfig --prefix="$_sdlpath" --libs`"
+		append_var SDL_LIBS "`pkg-config --libs sdl2`"
 	fi
-	_sdlversion=`$_sdlconfig --version`
+	_sdlversion=`pkg-config --modversion sdl2`
 
 	echocheck "SDL"
 	_sdl=no
-- 
2.26.2

