Give server more time to start

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>

diff -Nurd -X.diff_ignore freeciv/client/connectdlg_common.c freeciv/client/connectdlg_common.c
--- freeciv/client/connectdlg_common.c	2011-07-30 00:24:42.000000000 +0300
+++ freeciv/client/connectdlg_common.c	2012-05-14 23:31:13.000000000 +0300
@@ -60,7 +60,7 @@
 #include "tilespec.h"
 
 #define WAIT_BETWEEN_TRIES 100000 /* usecs */ 
-#define NUMBER_OF_TRIES 500
+#define NUMBER_OF_TRIES 10000
   
 #ifdef WIN32_NATIVE
 /* FIXME: this is referenced directly in gui-win32/connectdlg.c. */
