From 6276f1388819e6f03e61133494e7c9d35c785e17 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 8 Feb 2019 19:11:19 +0100
Subject: [PATCH] Use pkg-config to find sdl
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>

diff -Nurd Maelstrom-3.0.7/configure.in Maelstrom-3.0.7/configure.in
--- Maelstrom-3.0.7/configure.in	2021-02-04 19:17:55.000000000 +0200
+++ Maelstrom-3.0.7/configure.in	2022-08-24 23:12:31.161825755 +0300
@@ -47,10 +47,7 @@
 
 dnl Check for SDL
 SDL_VERSION=2.0.14
-AM_PATH_SDL2($SDL_VERSION,
-            :,
-            AC_MSG_ERROR([*** SDL version $SDL_VERSION not found!])
-)
+PKG_CHECK_MODULES([SDL], [sdl2 >= $SDL_VERSION], [HAVE_SDL=yes], [AC_MSG_ERROR([*** SDL version $SDL_VERSION not found!])])
 CFLAGS="$CFLAGS $SDL_CFLAGS"
 LIBS="$LIBS $SDL_LIBS"
 
