From b051343fcac17227369bc4c7e0b0b00811500e24 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Tue, 12 Feb 2019 15:06:01 +0100
Subject: [PATCH] Fix install dir
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [Configuration]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 Makefile.am  | 2 +-
 configure.in | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 4a9e949..a3194fc 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -65,7 +65,7 @@ DIST_SUBDIRS = $(SUBDIRS) Images Docs
 
 # Special install rule for the game
 install:
-	make install_gamedata target=@GAME_INSTALLDIR@
+	make install_gamedata target=$(DESTDIR)@GAME_INSTALLDIR@
 
 install_gamedata:
 	sh mkinstalldirs $(target)/
diff --git a/configure.in b/configure.in
index a6f76ac..e158dd2 100644
--- a/configure.in
+++ b/configure.in
@@ -105,6 +105,7 @@ case "$target" in
         GAME_INSTALLDIR="\$(prefix)/games/$PACKAGE"
         ;;
 esac
+GAME_INSTALLDIR="\$(datadir)/$PACKAGE"
 AC_SUBST(GAME_INSTALLDIR)
 CFLAGS="$CFLAGS -DLIBDIR=\\\"$GAME_INSTALLDIR\\\""
 
-- 
2.20.1

